src/index.jsx

```diff
@@ -2,25 +2,39 @@ import 'react-hot-loader/root'; // This needs to be imported before React.
 import React from 'react';
 import { render } from 'react-dom';
 import * as Sentry from '@sentry/react';
+import axios from 'axios';
 import { get } from 'lodash-es';
 
 import 'normalize.css';
 import '../fonts/fonts.css';
 import './styles/globalStyles.css';
 
-import { sentryDsn } from './constants/apiKeys';
 import pjson from '../package.json';
 import App from './App';
 
-if (!__DEV__) {
-  Sentry.init({
-    dsn: sentryDsn,
-    release: `frontend@${get(pjson, 'release')}`,
-  });
-}
+axios
+  .get(`${__houston_url__}/api/v1/site-settings/main/sentryDsn`)
+  .then(response => {
+    if (__DEV__) return;
 
-const root = document.getElementById('root');
+    const sentryDsn = get(response, [
```

Comment:
I don't know enough about sentry to know how to determine whether this works as expected.